### DISCUSSION TITLE
Units not selectable anymore after Update

### DISCUSSION THREAD

**Discussion by marcus-at-localhost** [atrocore/atrocore#668] [Q&A] [answered]
Hi, I updated to Core 1.5.37 and PIM 1.7.15 and now the entries in the `Unit` dropdown are gone.

![image](https://user-images.githubusercontent.com/1337223/223092010-fc01571f-4a04-4c60-9a40-3f404a4bfb6b.png)

Is there something I can do/check? Everything looks fine as far as I can tell.

Update Log
```
- Updating symfony/workflow (v5.4.19 => v5.4.21): 
- Updating symfony/event-dispatcher (v5.4.19 => v5.4.21): 
- Updating atrocore/core (1.5.22 => 1.5.37): 
- Updating atrocore/pim (1.6.31 => 1.7.15): 
- Updating atrocore/dam (1.4.20 => 1.5.3): 
- Updating atrocore/export-feeds (1.6.27 => 1.6.33): 
- Updating atrocore/import-feeds (1.4.32 => 1.4.42): 
```

PS: is it better to file this as an issue in the atrocore or atropim repo? I'm always unsure ^^

**Comment by rratsun** [comment]
Hello,
We try to react on issues and discussions the same. It does not matter where you create and describe the problem. 
About problem: I created an internal issue and we will fix the problem very soon.

**Reply by marcus-at-localhost** [reply]
@rratsun I tried to debug it, but didn't get too far. I compared to the data in the demo.


yours (API response)
![image](https://user-images.githubusercontent.com/1337223/224267710-7a5783f2-459c-4ed1-acb4-63c90161bac2.png)

my (API response)
![image](https://user-images.githubusercontent.com/1337223/224268183-d8136cb2-cb13-4a3e-8bb6-b646a5c07192.png)


yours:
![image](https://user-images.githubusercontent.com/1337223/224267498-cf5f50a8-65f4-474e-b934-59066ef743a8.png)


my:
![image](https://user-images.githubusercontent.com/1337223/224267335-0501b094-b95a-42d4-9de5-a0a3ff90587b.png)


API responses seem to be similar, but somewhere the transformation of the date goes off
I'm not sure yet how to step through the compiled code, but I think it happened here somewhere:
https://github.com/atrocore/atrocore/blob/85dcdf6406eb22e3c329cc2f33cebdcc5e44ae6b/client/src/views/fields/unit.js#L99-L112

I checked this object `this.params.measure`

![image](https://user-images.githubusercontent.com/1337223/224272837-bcb25b2b-da45-4d50-af5d-ae17feb8deab.png)

And it looked fine on your side, but now I realize - this shouldn't be "Length" on my side, but "Temperatur" or something like that.

And verifying with your Demo, it has the same issue, just that the Length Measure exists and therefore is pulled and it my case it doesn't exist and stays empty. 

![image](https://user-images.githubusercontent.com/1337223/224273472-391b8db9-384b-4929-be37-10004a048809.png)

I hope that helps, or maybe you already know :)

Best, marcus

**Comment by rratsun** [comment, marked as answer]
Fixed at PIM 1.7.18

**Reply by marcus-at-localhost** [reply]
Fascinating where the actual fix happened :D. Thank you!
